src/libraries/System.IO.Ports/src/CompatibilitySuppressions.xml

```diff
@@ -0,0 +1,86 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Suppressions xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
+  <Suppression>
+    <DiagnosticId>CP0001</DiagnosticId>
+    <Left>lib/netstandard2.0/System.IO.Ports.dll</Left>
+    <Right>lib/net461/System.IO.Ports.dll</Right>
+  </Suppression>
+  <Suppression>
+    <DiagnosticId>CP0001</DiagnosticId>
+    <Target>T:System.IO.Ports.Handshake</Target>
+    <Left>lib/net461/System.IO.Ports.dll</Left>
+    <Right>runtimes/unix/lib/netstandard2.0/System.IO.Ports.dll</Right>
+  </Suppression>
+  <Suppression>
+    <DiagnosticId>CP0001</DiagnosticId>
+    <Target>T:System.IO.Ports.Parity</Target>
+    <Left>lib/net461/System.IO.Ports.dll</Left>
+    <Right>runtimes/unix/lib/netstandard2.0/System.IO.Ports.dll</Right>
```

Comment:
@Anipik we should really look into this, I can't understand why we are comparing a net461 asset as a compile asset against a Unix runtime specific asset.

Reply:
When I talked with @Anipik yesterday he mentioned that this might be a NuGet bug when calling into their APIs.

Reply:
Do we have an issue opened yet? Can we workaround it?

Reply:
@Anipik is the expert on this. I'll defer to him.